src/cli/command/server.rs

```diff
@@ -0,0 +1,130 @@
+use std::{
+	env,
+	fs,
+    default::Default,
+	fs::OpenOptions,
+	path::Path,
+	collections::HashMap,
+	sync::{Arc, RwLock}
+};
+use daemonize::Daemonize;
+use clap::{ArgMatches};
+use sysexits::ExitCode;
+use actix_web::{
+	middleware, web, App, HttpResponse, HttpServer
+};
+use crate::{
+    http,
+	errors::RvError,
+	EXIT_CODE_OK, EXIT_CODE_INSUFFICIENT_PARAMS, EXIT_CODE_LOAD_CONFIG_FAILURE,
+	storage::{physical, barrier_aes_gcm},
+	core::Core,
+};
+
+pub fn main(config: &str) -> Result<(), RvError> {
+	env::set_var("RUST_LOG", "debug");
+	env_logger::init();
+
+	let work_dir = "/tmp/rusty_vault/";
+
+    log::info!("config: {}, work_dir: {}", config, work_dir);
+
+	if !Path::new(work_dir).exists() {
+        log::info!("create work_dir: {}", work_dir);
+		fs::create_dir_all(work_dir)?;
+	}
+
+	let dev = true;
+
+	if !dev {
+		// start daemon
+		let log_path = format!("{}/info.log", work_dir);
+		let pid_path = format!("{}/rusty_vault.pid", work_dir);
+
+		let log_file = OpenOptions::new()
+			.read(true)
+			.write(true)
+			.create(true)
+			.open(log_path)
+			.unwrap();
+
+		let daemonize = Daemonize::new()
+			.pid_file(pid_path.clone())
+			.chown_pid_file(true)
+			.working_directory(work_dir)
+			.user("jinjiu")
+			.group("staff")
```

Comment:
It seems user and group are not mandatory, so they can be omitted.

Reply:
done